Lib/test/test_sys.py

```diff
@@ -2114,11 +2114,18 @@ def audit_hook(event, arg):
         script = '''
 print("Remote script executed successfully!")
 '''
+        remote_exec_event_triggered = False
+        def audit_hook(event, arg):
+            if event == "remote_exec":
+                nonlocal remote_exec_event_triggered
+                remote_exec_event_triggered = True
```

Comment:
It would be better to save `arg`, not just `True`. And compare `arg` with the expected value.